console/Router.php

```diff
@@ -34,10 +36,14 @@ public static function parse($request = null) {
 			while ($args) {
 				$arg = array_shift($args);
 				if (preg_match('/^-(?P<key>[a-zA-Z0-9])$/i', $arg, $match)) {
+					$key = Inflector::camelize($match['key'], false);
+					$params[$key] = true;
 					$params[$match['key']] = true;
```

Comment:
If possible make this one line (only if the same ca be done for line 46&47 too without exceeding line length limit -- for symmetry). 
```$params[$key] = $params[$match['key'] = true```